README.md

```diff
@@ -178,26 +180,25 @@ $a[3] = 'd';   // or use $a->offsetSet(3, 'd') method
 $a->toArray(); // [0 => 'a', 1 => 'b', 2 => 'c', 3 => 'd']
 
 print $a[1]; // 'b'
-// or use corresponding method
+// or use the corresponding method
 print $a->offsetGet(1); // 'b'
 ```
 
-> **NOTE:** Further will be used the `A` namespace alias to shorten usage examples, but keep in mind
-  that the usage are suitable for both `MutableArray` and `ImmutableArray` in this case.
-
 ### Chaining
 
-Use chaining methods for fast usage:
+*NOTE: The following methods and principles apply to ImmutableArray and MutableArray alike*
```

Comment:
Could you add dot at the end and **`** quotes around class names:
*...apply to `ImmutableArray` and `MutableArray` alike.*